github/repos.go

```diff
@@ -201,6 +201,14 @@ type RepositoryListByOrgOptions struct {
 	// forks, sources, member. Default is "all".
 	Type string `url:"type,omitempty"`
 
+	// How to sort the repository list. Can be one of created, updated, pushed,
+	// full_name. Default: full_name
```

Comment:
It looks like the docs say that `created` is the default option. Could you please update your comment?